data/allSolidTumors_disease_ontology.json

```diff
@@ -0,0 +1,2480 @@
+        {
+          "class": "ElementOf",
+          "target": "all solid tumors"
+        }
+      ],
+      "name": "bladder urothelial carcinoma",
+      "source": "oncotree",
+      "sourceId": "blca"
+    },
+    "srcbc": {
+      "links": [
+        {
+          "class": "ElementOf",
+          "target": "all solid tumors"
+        }
+      ],
+      "name": "plasmacytoid/signet ring cell bladder carcinoma",
+      "source": "oncotree",
+      "sourceId": "srcbc"
+    },
+    "urca": {
+      "links": [
+        {
+          "class": "ElementOf",
+          "target": "all solid tumors"
+        }
+      ],
+      "name": "urachal carcinoma",
+      "source": "oncotree",
+      "sourceId": "urca"
+    },
+    "chbl": {
+      "links": [
+        {
+          "class": "ElementOf",
+          "target": "all solid tumors"
+        }
+      ],
+      "name": "chondroblastoma",
+      "source": "oncotree",
+      "sourceId": "chbl"
+    },
+    "gctb": {
+      "links": [
+        {
+          "class": "ElementOf",
+          "target": "all solid tumors"
+        }
+      ],
+      "name": "giant cell tumor of bone",
+      "source": "oncotree",
+      "sourceId": "gctb"
+    },
+    "armm": {
+      "links": [
+        {
+          "class": "ElementOf",
+          "target": "all solid tumors"
+        }
+      ],
+      "name": "anorectal mucosal melanoma",
+      "source": "oncotree",
+      "sourceId": "armm"
+    },
+    "c42598": {
+      "links": [
+        {
+          "class": "ElementOf",
+          "target": "all solid tumors"
+        }
+      ],
+      "name": "low-grade appendiceal mucinous neoplasm",
+      "source": "ncit",
+      "sourceId": "c42598"
+    },
+    "tac": {
+      "links": [
+        {
+          "class": "ElementOf",
+          "target": "all solid tumors"
+        }
+      ],
+      "name": "tubular adenoma of the colon",
+      "source": "oncotree",
+      "sourceId": "tac"
+    },
+    "lcis": {
+      "links": [
+        {
+          "class": "ElementOf",
+          "target": "all solid tumors"
+        }
+      ],
+      "name": "breast lobular carcinoma in situ",
+      "source": "oncotree",
+      "sourceId": "lcis"
+    },
+    "brca": {
+      "links": [
+        {
+          "class": "ElementOf",
+          "target": "all solid tumors"
+        }
+      ],
+      "name": "invasive breast carcinoma",
+      "source": "oncotree",
+      "sourceId": "brca"
+    },
+    "difg": {
+      "links": [
+        {
+          "class": "ElementOf",
+          "target": "all solid tumors"
+        }
+      ],
+      "name": "diffuse glioma",
+      "source": "oncotree",
+      "sourceId": "difg"
+    },
+    "bgct": {
+      "links": [
+        {
+          "class": "ElementOf",
+          "target": "all solid tumors"
+        }
+      ],
+      "name": "germ cell tumor, brain",
+      "source": "oncotree",
+      "sourceId": "bgct"
+    },
+    "pint": {
+      "links": [
+        {
+          "class": "ElementOf",
```

Comment:
As discussed in zoom can you change these "ElementOf" to "SubclassOF"? they were entered incorrectly by the analyst who manually added them originally

Reply:
Sure. I misunderstood that part.